global.json

```diff
@@ -2,5 +2,10 @@
     "projects": [
         "src",
         "submodules/FileSystem/src"
-    ]
+    ],
+    "sdk": {
+        "version": "1.0.0-beta6-12207",
```

Comment:
I don't even know how it got there, and i'm pretty sure I saw it and removed it at one point :P. I'll remove it now.

Also, I'll double-check that VS isn't re-adding it somehow and ensure it's removed before final merge if it somehow reappears :)